be2-scala/README.md

```diff
@@ -68,8 +70,38 @@ The project relies on several sbt dependencies (external libraries) :
 
 ---
 
-
-
 ## Coding convention
 
 Our coding guidelines can be found [here](https://docs.scala-lang.org/style/).
+
+## Install java and sbt on linux and Mac
+
+[skdman](https://sdkman.io/) does a pretty good job for installing the required
+environment:
+
+```sh
+# install sdkman
+curl -s "https://get.sdkman.io" | bash 
+# install java zulu 11
+sdk install java 11.0.18-zulu
```

Comment:
are you sure? 
On popdemo we install OpenJDK 17.

Reply:
The only example I had at that point was the CI, I blindly followed what is done there 🙈

Reply:
But this is fine, it will run. We update this one and the CI in the future.